dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/table/model/AnalyticsTable.java

```diff
@@ -252,9 +253,17 @@ public AnalyticsTablePartition getLatestTablePartition() {
         .orElse(null);
   }
 
+  // implement toString method
+
   @Override
   public String toString() {
-    return "[Table name: " + getName() + ", partitions: " + tablePartitions + "]";
```

Comment:
This causes StackOverFlow exception as the `tablePartitions` will trigger infinite loop calls to `AnalyticsTable.toString()` method.